### ISSUE TITLE
Wobble animation in carousel

### ISSUE THREAD

**Issue by erbanach** [metafizzy/flickity#837]
There is a slight wobble animation in my Flickity carousel.

Here is my Codepen with my code: https://codepen.io/erbanach/pen/xayXGG

I have added some custom styling, but I am not seeing anything specific that could be causing this, unless I am missing something. Is this an option that needs to be turned off? Any assistance would be helpful! Thank you.

**Comment by bdcoder2** [comment]
I changed the options to:
```
  autoPlay: 4000,
  draggable: false,
  wrapAround: true
```

And everything seemed to work well - using Version 68.0.3440.106 (Official Build) (64-bit)

**Comment by desandro** [comment]
Yeah, your settings for selectedAttraction and friction are causing the wobble. If you want a slower transition speed, I would use

``` js
  selectedAttraction: 0.01,
  friction: 0.3
```
